src/components/measureLanding/MeasureLanding.tsx

```diff
@@ -7,10 +7,10 @@ import NewMeasure from "../newMeasure/NewMeasure";
 export function MeasureRoutes() {
```

Comment:
Thoughts on renaming this file, maybe `MeasureRoutes` or `MeasureRouter`? It was originally a functional component, but has since been repurposed to a router.

Reply:
okay, I'll have to create another PR for component name changes. This is a multi-repository fix and I merged other PRs without thinking it is going to break dev server.